### ISSUE TITLE
can't find source code when using Parallel Computing

### ISSUE THREAD

**Issue by magdalenasabat** [JorritMontijn/zetatest#3]
Hi! 

I'm using MATLAB 2021a on Ubuntu 20.04. 

When an a tive parallel pool is available when running `zetatest.m` some dependencies are missing inside parfor loops. I might be setting my paths wrong, so this is either a bug or a question :) 

```
Error using getMultiScaleDeriv (line 92)
The source code (/~/toolbox-zetatest/dependencies/getMultiScaleDeriv.m) for the parfor-loop that is trying to execute on the
worker could not be found.

Error in zetatest (line 323)
		[vecRate,sRate] = getMultiScaleDeriv(vecSpikeT,vecRealDiff,[],[],[],intPlot,dblMeanRate,dblUseMaxDur);

Error in Script_zeta_select_neurons (line 134)
        [dblZetaP, vecLatencies,sZETA,sRate] = zetatest(vecSpikeTimes,matEventTimes,dblUseMaxDur,intResampNum , intResampNum,intPlot,intLatencyPeaks,boolDirectQuantile)

Caused by:
    Undefined function 'calcSingleMSD_mex' for input arguments of type 'double'.
    Error using remoteParallelFunction (line 84)
    Worker unable to find file.
        Undefined function 'calcSingleMSD_mex' for input arguments of type 'double'.
```

**Comment by JorritMontijn** [comment]
Hi Magdalena,

Thank you for your message. Apologies, this is indeed a bug - the compiled mex-file only works on windows and I made no checks to see if it would actually run. I've added a backup routine in case the compiled mex doesn't work, so this bug should be fixed now in v3.5.1. Let me know if you still have problems.

Best wishes,
Jorrit

**Comment by JorritMontijn** [comment]
By the way, you could also try compiling the calcSingleMSD() function embedded in getMultiScaleDeriv.m yourself and replacing the calcSingleMSD_mex.mexw64 file for windows with your locally compiled version for ubuntu. It's a bit faster after compilation, but it doesn't make a huge difference, so don't spend too much time on it if you don't want to.
